Containerfile

```diff
@@ -0,0 +1,31 @@
+ARG base_image=centos:8
+ARG package_manager
+FROM $base_image
+
+ENV package_manager ${package_manager:-yum -y}
+RUN $package_manager -y install gcc git krb5-devel python3-devel python3-pip rpm-build && $package_manager -y clean all
```

Comment:
Yes, that's true, but the reason `clean all` is called here, is to avoid having that metadata cache baked into the image.

And the reason I choose to have 2 steps for installation is that this section is for the dependencies of the script, while the following one is for dependencies required to run the `%prep` sections. By keeping these two separate it was faster to rebuild the image, as the script dependencies rarely changed (so a cached layer could be used), but I needed to have several iterations already to update the second list (and I expect to have several more as we convert more projects).

Unless you feel strong about it, I would leave it like it is now, and maybe merge the two steps once the list of dependencies becomes stable.